app/services/IdentityService.scala

```diff
@@ -26,7 +26,14 @@ class IdentityService(identityApiClient: => IdentityApiClient) extends LazyLoggi
 
   def doesUserExist(email: String): Future[Boolean] =
     identityApiClient.userLookupByEmail(email).map { response =>
-      (response.json \ "user" \ "id").asOpt[String].isDefined
+      response.status match {
+        case status if status == Status.OK =>  (response.json \ "user" \ "id").asOpt[String].isDefined
+        case status => {
+          logger.error(s"ID API failed on email check with HTTP status $status")
```

Comment:
Presumably we need to restrict this case to certain codes though (e.g. 404s are presumably not an error, because this just indicates that the user doesn't have Identity yet)